cs-minibot-platform-src/src/main/resources/public/js/view.js

```diff
@@ -260,6 +275,43 @@ function setupGridLines(scale, xOffset, yOffset) {
         gridContainer.addChild(lines_x[i]);
     }
 }
+function fillOccupancyMatrix(scale, xOffset, yOffset) {
+    for(var i = 0; i < occupancyMatrix.length; i++) {
+        for(var j = 0; j < occupancyMatrix[0].length; j++) {
+
+                if(path[i][j] == 1) {
```

Comment:
add one line comments to the function describing what these cases mean- not really clear to me